eZ/Bundle/EzPublishCoreBundle/Command/UpdateTimestampsToUTCCommand.php

```diff
@@ -287,7 +289,8 @@ protected function processTimestamps($offset, $limit)
             $dateTimeInUTC->setTimestamp($timestamp);
             $newTimestamp = $this->convertToUtcTimestamp($timestamp);
 
-            if (!$this->dryRun) {
+            //failsafe for int field limitation (dates/datetimes after 01/19/2038 @ 4:14am (UTC))
+            if ($newTimestamp <= 2147483647 && !$this->dryRun) {
```

Comment:
Maybe not the most elegant option, but still better than a hardcoded value 😉 
```
PHP_INT_SIZE == 4 ? PHP_INT_MAX : PHP_INT_MAX>>32;
```

Reply:
Thanks for pointing it out

Reply:
guys... `PHP_INT_SIZE` should correspond to `PHP_INT_MAX`. Why not use `PHP_INT_MAX` constant instead? Are there some date/time functions constraints preventing that?

Reply:
@alongosz the problem is that for MySQL max int value is like `PHP_INT_MAX` for 32b env. So it doesn't make sense to use `PHP_INT_MAX` in case of 64b envs.

Reply:
As long as we use INT and not BIGINT for this column, it is ineed limited to same as 32bit php: https://dev.mysql.com/doc/refman/8.0/en/integer-types.html